### PR TITLE
+aus_seasons()

### PR DESCRIPTION
**Pull request by carlosyanez**
Hi !

I would like to suggest a new , very simple helper function: aus_seasons().
This function takes a vector with dates and returns a vector with the corresponding season names - following the Australian seasons definitions (i.e. month aligned rather than solstice-equinox).

The package comes with both 4 seasons and Top End, "tropical" wet/dry seasons.

```
sample_dates <- c("2022-01-01","2018-10-04","2016-07-20")
# temperate seasons
aus_seasons(sample_dates,"temperate")
#> [1] "Summer" "Spring" "Winter"
# tropical seasons
aus_seasons(sample_dates,"tropical")
#> [1] "Wet Season" "Dry Season" "Dry Season"

```

Kind Regards,
Carlos

### CODE REVIEW

**Comment by MattCowgill** [comment]
Looks good to me. 

@wfmackey / @daviddiviny / @mrjoh3 / anyone else: any thoughts?

**Comment by MattCowgill** [comment]
Thanks for contributing @carlosyanez !

File: R/aus_seasons.R [hunk 1]
```diff
@@ -0,0 +1,47 @@
+library(lubridate)
```
Comment (mrjoh3) [inline R/aus_seasons.R:1]: need to remove `library(lubridate)` from the file. Instead add `#' imports lubridate` to the roxygen header.

**Review by mrjoh3 — CHANGES_REQUESTED** [review]
great idea and the code looks good. Just need to make one change importing the lubridate library

File: R/aus_seasons.R (hunk 1 above)
Comment (carlosyanez) [reply R/aus_seasons.R]: Thanks for noticing the "leftover" `library(lubridate)`.  I've corrected it now.

**Review by MattCowgill — APPROVED** [review]

**Review by wfmackey — APPROVED** [review]

**Comment by wfmackey** [comment]
Nice work, thanks for contributing @carlosyanez!

**Pull request merged**